pywbemtools/pywbemcli/mockscripts/__init__.py

```diff
@@ -213,7 +213,7 @@ def script_error(file_path, exc):
     Return a MockScriptError exception object with the current traceback in its
     exception message, ready to be displayed.
     """
-    if isinstance(exc, pywbem.Error):
+    if isinstance(exc, pywbem.Error) and False:
```

Comment:
I do not understand this statement with the and False added.  I am missing something here.

Reply:
This is WIP. The statement temporarily shows `pywbem.Error` exceptions with traceback.

Reply:
Ahhha.  I was having real problems trying to determine if that and False had some other reason.

Reply:
I have added an according TODO to remove it again.